src/Symfony/Bridge/Twig/Extension/SecurityExtension.php

```diff
@@ -21,13 +24,43 @@
  */
 class SecurityExtension extends AbstractExtension
 {
+    private $securityChecker;
+
+    public function __construct(AuthorizationCheckerInterface $securityChecker = null)
+    {
+        if (null !== $securityChecker) {
+            @trigger_error(sprintf('The $securityChecker parameter is deprecated since 3.4 and will be removed in 4.0. Use %s instead.', SecurityRuntime::class), E_USER_DEPRECATED);
```

Comment:
We could add an `@final since version 4.1` annotation in the class docblock, that would avoid the need for triggering by method.

Reply:
Except for the constructor arg in fact, this one is not about inheritance but direct instantiation. It should be kept